index.js

```diff
@@ -212,6 +215,25 @@ Feed.prototype.replicate = function (initiator, opts) {
   return stream
 }
 
+Feed.prototype.setDownloading = function (downloading) {
+  this.downloading = downloading
+  this._downloadingSet = true
```

Comment:
If downloading is `false` should this still be set to `true`?

Reply:
yes, just whether it's been set or not